### DISCUSSION TITLE
使用不稳定，之前都正常，今天就变成这样了

### DISCUSSION THREAD

**Discussion by DavidXie03** [acheong08/ChatGPT#978] [Q&A]
![QQ截图20230302115631](https://user-images.githubusercontent.com/62877238/222327550-652e253e-3395-406c-8a8f-a21eca6dcdfa.png)

**Comment by acheong08** [comment]
Set `export CHATGPT_BASE_URL=https://china.duti.tech/`

**Reply by DavidXie03** [reply]
请问在哪里设置呢

**Reply by acheong08** [reply]
```py
import os

os.environ["CHATGPT_BASE_URL"] = "https://china.duti.tech/"
```

**Reply by DavidXie03** [reply]
设置了以后好像没啥变化，这个网站也显示404 page not found

**Reply by acheong08** [reply]
Maybe access token expired

**Reply by acheong08** [reply]
> 设置了以后好像没啥变化，这个网站也显示404 page not found

It's supposed to. It's used as an API

**Reply by DavidXie03** [reply]
好的 大概持续多久会过期呢

**Reply by DavidXie03** [reply]
非常感谢 改了session_token以后就解决了

**Comment by jhcao23** [comment]
why can't we set `os.environ["CHATGPT_BASE_URL"] = "https://chat.openai.com/"`? I tried but `chat.openai.com` never work, how come?
